sdk/clientcore/core/src/main/java/io/clientcore/core/annotations/ReturnType.java

```diff
@@ -21,5 +21,10 @@ public enum ReturnType {
     /**
      * Server Sent Event operation return type.
      */
-    SERVER_SENT_EVENT
+    SERVER_SENT_EVENT,
+
+    /**
+     * Long-running operation return type.
+     */
+    LONG_RUNNING_OPERATION;
```

Comment:
Are the changes other than AzureCloud intentional?

Reply:
Yes, these were some misc changes that were sitting in my local branch. So, pushed them along with the change to include AzureCloud.